app/ui-message/client/message.html

```diff
@@ -55,25 +57,27 @@
 					<time class="time" title='{{date}} {{time}}' datetime='{{date}} {{time}}'>{{time}}</time>
 					{{#if showTranslated}}
 						<span class="translated">
```

Comment:
Are these span elements still relevant?

Reply:
I don't know if this is important... is more safe to stay that until next rewrite of message...